### PR TITLE
MBM-301/302/303/304 Earth Defensive spell

### PR DESCRIPTION
**Pull request by insertStringHere**

### CODE REVIEW

**Review by MattKirchh — CHANGES_REQUESTED** [review]
The arm animation happens even if no spell is selected, is there a way that could be checked before updating the animation?

File: src/main/resources/assets/minebound/models/item/magic_earth_d1.json
Comment (insertStringHere) [inline src/main/resources/assets/minebound/models/item/magic_earth_d1.json:1]: Delete models

File: src/main/java/com/mineboundteam/minebound/magic/DefensiveSpells/EarthDefensiveSpell.java [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+import net.minecraftforge.common.ForgeConfigSpec.BooleanValue;
+import net.minecraftforge.common.ForgeConfigSpec.DoubleValue;
+import net.minecraftforge.common.ForgeConfigSpec.EnumValue;
+import net.minecraftforge.common.ForgeConfigSpec.IntValue;
+import net.minecraftforge.event.ForgeEventFactory;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.event.config.ModConfigEvent;
+
+@Mod.EventBusSubscriber(modid = MineBound.MOD_ID)
+public class EarthDefensiveSpell extends ActiveSpellItem {
```
Comment (MattKirchh) [inline src/main/java/com/mineboundteam/minebound/magic/DefensiveSpells/EarthDefensiveSpell.java:39]: Is this `EventBusSubscriber` needed? There's no event being subscribed to in this class.

Comment (insertStringHere) [reply src/main/java/com/mineboundteam/minebound/magic/DefensiveSpells/EarthDefensiveSpell.java]: At one point there were; removed

**Review by MattKirchh — APPROVED** [review]

**Review by YonatanWN — APPROVED** [review]

**Pull request merged**